src/js/components/Box/stories/TEMP.js

```diff
@@ -0,0 +1,177 @@
+  },
+  {
+    background: 'image',
+    label: { value: 'image', context: 'theme string' },
+  },
+  {
+    background: {
+      color: 'light-1',
+    },
+    label: { value: 'hover', context: 'over me' },
+    onClick: () => {},
+    hoverIndicator: 'stack',
+  },
+  {
+    background: `linear-gradient(
+      155deg,
+      hsl(147deg 100% 72%) 0%,
+      hsl(223deg 100% 72%) 50%,
+      hsl(299deg 100% 72%) 100%
+    );`,
+    label: { value: 'gradient', context: 'prop string' },
+  },
+  {
+    background: 'gradient',
+    label: { value: 'gradient', context: 'theme string' },
+  },
+  {
+    background: { dark: 'accent-2', light: 'accent-3' },
+    label: { value: 'dark-light object', context: 'prop object' },
+  },
+  {
+    background: 'dark-light-image',
+    label: { value: 'dark-light object', context: 'theme object' },
+  },
+  {
+    background: {
+      image: 'dark-light-image',
+      dark: true,
+      color: { dark: 'black', light: 'gold' },
+    },
+    label: { value: 'dark-light object force dark', context: 'theme object' },
+  },
+  {
+    background: {
+      color: 'neutral-2',
+      image: `url(https://images.unsplash.com/photo-1653023500770-3a3b64a1b4c4?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1632&q=80)`,
+      position: 'top-left',
+    },
+    label: { value: 'object', context: 'prop object' },
+  },
+  {
+    background: 'object',
+    label: { value: 'object', context: 'theme object' },
+  },
+  {
+    background: { image: 'gradient', rotate: 180 },
+    label: { value: 'rotate', context: 'prop object' },
+  },
+  {
+    background: {
+      image: 'gradient',
+      rotate: -90,
+      clip: 'text',
+    },
+    label: {
+      value: <Text size="xxlarge">clip text</Text>,
+      context: <Text size="medium">prop object</Text>,
+    },
+  },
+  {
+    background: {
+      image: 'image',
+      rotate: -90,
+      clip: 'text',
+    },
+    label: {
+      value: <Text size="xxlarge">clip text</Text>,
+      context: <Text size="medium">prop object</Text>,
+    },
+  },
+];
+
+const Swatch = ({ label, ...rest }) => (
+  <Box fill pad="medium" justify="center" round="small" {...rest}>
+    <Text weight="bold" size="large">
+      {label.value}
+    </Text>
+    <Text size="small">{label.context}</Text>
+  </Box>
+);
+
+export const BackgroundBoxTemp = () => (
+  // Uncomment <Grommet> lines when using outside of storybook
+  <Grommet
+    theme={myTheme}
+    full
+    background={`linear-gradient(
+      155deg,
+      hsl(0deg 0% 51%) 0%,
+      hsl(0deg 0% 76%) 50%,
+      hsl(0deg 0% 100%) 100%
+    )`}
+  >
+    <Grid columns="small" rows="small" gap="small" pad="large">
+      {backgroundSamples.map((sample, index) => (
+        <Swatch
+          // eslint-disable-next-line react/no-array-index-key
+          key={index}
+          background={sample.background}
+          label={sample.label}
+          animation={{ type: 'fadeIn', delay: index * 50 }}
```

Comment:
This is very nice on the human eye. Unfortunately, it doesn't play well with Chromatic as Chromatic doesn't wait for the animations to play. It just sees a blank grey screen. I think the visual regression testing trumps the interactive animation elegance and I think we should remove the animation.

Reply:
Do you see value in keeping this story outside of testing? If so, should we combine this with the other "Background" story? Keep this story separate, but rename to "Background from theme", "Themed backgrounds", other?

I was thinking we'd either remove this story from the commit or combine it with the existing background story. Thoughts?

Reply:
1) Removed animation for Chromatic snapshots, 2) put up a story which iterates through backgrounds in the currently selected theme, and 3) moved the custom theme story under the Custom Theme directory.